backend/cmd/worker/patient_validation.go

```diff
@@ -27,27 +28,32 @@ var AllowedOrganizationCategories = []string{"healthcare_provider", "research_in
 const PatientAlreadyExistCode = "PATIENT-001"
 const ExistingPatientDifferentFieldCode = "PATIENT-002"
 const OrganizationNotExistCode = "PATIENT-003"
-const FieldTooLongCode = "PATIENT-004"
-const RegExpNotMatchCode = "PATIENT-005"
-const OrganizationTypeCode = "PATIENT-006"
+const InvalidValueCode = "PATIENT-004"
```

Comment:
I think we might have collision with samples if we keep the same name `InvalidValueCode` because they will share package name (`main`). 

Maybe prefix with `Patient` -> `PatientInvalidValueCode`